index.html

```diff
@@ -510,6 +519,55 @@ <h3>Security and Privacy Considerations</h3>
 
 		</section>
 
+		<section id="audio-ua-behaviour">
+			<h2>User Agent Behaviours for Audiobooks</h2>
+
+			<p>This section will outline the expected user agent behaviours for implementation of audiobooks. For processing instructions, user agents should refer to the <a href="https://www.w3.org/TR/pub-manifest/#manifest-processing"><code>Processing a Manifest</code></a> section of the Publication Manifest specification, and conform to any behaviour described there. This section will outline any additional user agent behaviours for audiobooks.</p>
+
+			<p>All user agent behaviours described in this section are intended to provide implementors with guidance, not strict requirements. Behaviours in this document will be mainly taken from the <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note published by the working group.</p>
+
+			<section id="audio-ua-navigation">
+				<h3>Opening and Navigating the Contents of an Audiobook</h3>
+
+				<p>When a user requests for a User Agent to open an audiobook, and the manifest processes according to the rules laid out in <a href="https://www.w3.org/TR/pub-manifest/#manifest-processing">Publication Manifest</a>, it should be opened by the User Agent. The <a href="#audio-readingorder">Reading Order</a> or, if available, <a href="#audio-toc">Table of Contents</a> should be accessible to the user. If a non-audio resource is present in the Reading Order, the User Agent can choose to present it to the user or skip it.</p>
+
+				<p>Resources within the Reading Order and Resource list that are non-audio resources should have a method for display, if possible for the User Agent. If the content cannot be displayed in the User Agent for any reason, it is recommended that the User Agent inform the user that the content is present by displaying it in the table of contents (if available), and provide a method for accessing it (see the section on <a href="#audio-ua-packaging">Packaging</a>.</p>
+
+				<p>If an audiobook has a <a href="#audio-pep">Primary Entry Page</a>, the User Agent can choose to display that as the Table of Contents.</p>
+
+			</section>
+
+			<section id="audio-ua-playback">
+				<h3>Audiobook Playability</h3>
+
+				<p>As outlined in the <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note, an audiobook should be navigable in the User Agent. This means that a User Agent should provide methods for the user to move through the audiobook in a linear or non-linear fashion by either moving through the <a href="#audio-readingorder">Reading Order</a> seamlessly or by accessing the <a href="#audio-toc">Table of Contents</a>. The User Agent should also allow the user to move through individual audio files in short time increments.</p>
```

Comment:
Aren't these "musts"? Seems like the baseline for something calling itself a user agent.